File: CrimsonClubs/CrimsonClubs.Models/Dtos/EventDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CrimsonClubs.Models.Dtos
{
    public class EventDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Start { get; set; }
        public DateTime Finish { get; set; }
        public bool IsGroupEvent { get; set; }
        public List<ClubDto> Clubs { get; set; }
        public List<StatClubDto> ClubStats { get; set; }
        public List<StatGroupDto> GroupStats { get; set; }
        public List<UserStatsDto> UsersClubStats { get; set; }
        public List<UserStatsDto> UsersGroupStats { get; set; }

        public int ClubId { get; set; }
        public string ClubName { get; set; }
    }
}

File: Android/CrimsonClubs/app/src/main/java/crimsonclubs/uacs/android/crimsonclubs/EventDto.java
package crimsonclubs.uacs.android.crimsonclubs;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.time.LocalDateTime;
import java.time.ZonedDateTime;
import java.util.ArrayList;
import java.util.Date;

public class EventDto {

    @Expose
    @SerializedName("id")
    public int id;
    @Expose
    @SerializedName("name")
    public String name;
    @Expose
    @SerializedName("description")
    public String description;
    @Expose
    @SerializedName("start")
    public String start;
    @Expose
    @SerializedName("finish")
    public String finish;
    @Expose
    @SerializedName("isGroupEvent")
    public boolean isGroupEvent;
    @Expose
    @SerializedName("clubId")
    public int clubId;
    @Expose
    @SerializedName("clubName")
    public String clubName;
    @Expose
    @SerializedName("clubs")
    public ArrayList<ClubDto> clubs;
    @Expose
    @SerializedName("clubStats")
    public ArrayList<StatClubDto> clubStats;
    @Expose
    @SerializedName("groupStats")
    public ArrayList<StatGroupDto> groupStats;
    @Expose
    @SerializedName("usersClubStats")
    public ArrayList<UserStatsDto> usersClubStats;
    @Expose
    @SerializedName("usersGroupStats")
    public ArrayList<UserStatsDto> usersGroupStats;
}
